engine/execution/state/cargo/payload/view.go

```diff
@@ -0,0 +1,76 @@
+package payload
+
+import (
+	"sync"
+
+	"github.com/onflow/flow-go/engine/execution/state/cargo/storage"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type View interface {
+	Get(
+		height uint64,
+		blockID flow.Identifier,
+		key flow.RegisterID,
+	) (
+		flow.RegisterValue,
+		error,
+	)
+}
+
+// oracle stores views that are finalized in an storage
+type OracleView struct {
```

Comment:
why do you need this?  it does nothing.  just use Storage directly

Reply:
removed it